docs/tutorials/using-time-series-prediction.md

```diff
@@ -96,12 +96,37 @@ There are three types of the metric query:
 Now we only support prometheus as data source. We define the `MetricType` to orthogonal with the datasource. but now maybe some datasources do not support the metricType.
 
 ### Algorithm
-`Algorithm` define the algorithm type and params to do predict for the metric. Now there are two kinds of algorithms:
-
- - `dsp` is an algorithm to forcasting a time series, it is based on FFT(Fast Fourier Transform), it is good at predicting some time series with seasonality and periods.
- - `percentile` is an algorithm to estimate a time series, and find a recommended value to represent the past time series, it is based on exponentially-decaying weights historgram statistics. it is used to estimate a time series, it is not good at to predict a time sequences, although the percentile can output a time series predicted data, but it is all the same value. so if you want to predict a time sequences, dsp is a better choice.
+`Algorithm` defines the algorithm type and the corresponding parameters. Currently, crane provides two built-in prediction algorithms:
+ - `dsp` borrows some techniques from 'digital signal processing', such as FFT(Fast Fourier Transform), for periodicity identification and time series forecasting. It is good at predicting the time series that follow some cyclic patterns.
```

Comment:
Please add a new line between the list title and the list item.
If not, the list style not be effective.
<img width="789" alt="image" src="https://user-images.githubusercontent.com/35299017/167251987-e2e32ab5-01a7-4c18-aeb5-675c2dd770e2.png">